src/components/sheet/SlackSheet.android.js

```diff
@@ -27,10 +27,13 @@ const Container = styled(Centered).attrs({ direction: 'column' })`
   position: absolute;
   border-top-left-radius: 20;
   border-top-right-radius: 20;
-  top: ${({ contentHeight, additionalTopPadding }) =>
-    contentHeight && additionalTopPadding
-      ? deviceUtils.dimensions.height - contentHeight
-      : 0};
+  ${({ deferredHeight, contentHeight, additionalTopPadding }) =>
```

Comment:
can we make it less trashy?

Reply:
extract to some function or use some meaningful names? Maybe I'm just a bit tired, but I got totally lost. 

However, at the same time I know we have already done a ton of more hacky code so idc too much

Reply:
haha I'd love to, any suggestions on how?